en/computer_vision/collision_prevention.md

```diff
@@ -196,10 +209,70 @@ make px4_sitl gz_x500_lidar_2d
 
 Next, adjust the relevant parameters to the appropriate values and add arbitrary obstacles to your simulation world to test the collision prevention functionality.
 
-The diagram below shows how the simulation looks when viewed in Gazebo.
+The diagram below shows a simulation of collision prevention as viewed in Gazebo.
 
 ![RViz image of collision detection using the x500_lidar_2d model in Gazebo](../../assets/simulation/gazebo/vehicles/x500_lidar_2d_viz.png)
 
-<!-- PR companion collision prevention (initial): https://github.com/PX4/PX4-Autopilot/pull/10785 -->
-<!-- PR for FC sensor collision prevention: https://github.com/PX4/PX4-Autopilot/pull/12179 -->
-<!-- using rangefinder? -->
+## Sensor Data Overview (Implementation Details)
+
```

Comment:
FYI, reorganized this to state more clearly what we do first, then how the rangefinder/obstacle/companion cases differ.